src/Goteo/Library/Forms/Model/ProjectPostForm.php

```diff
@@ -37,7 +37,10 @@ public function createForm() {
                 'label' => 'regular-title',
                 'constraints' => array(
                     new Constraints\NotBlank(),
-                    new Constraints\Length(array('min' => 4)),
+                    new Constraints\Length([
+                        'min' => 4,
+                        'allowEmptyString' => true,
+                    ]),
```

Comment:
`allowEmptyString` is now a mandatory option when using min option in `Length` constraint. To preserve the current behavior, it must be set to true.